pkg/option/config.go

```diff
@@ -384,6 +384,12 @@ const (
 	CTMapEntriesGlobalTCPName    = "bpf-ct-global-tcp-max"
 	CTMapEntriesGlobalAnyName    = "bpf-ct-global-any-max"
 
+	// NATMapEntriesGlobalDefault holds the default size of the NAT map
+	NATMapEntriesGlobalDefault = int((CTMapEntriesGlobalTCPDefault + CTMapEntriesGlobalAnyDefault) * 2 / 3)
```

Comment:
Why the `*2/3`?

Reply:
@borkmann but it should be as a comment here not only in the comit

Reply:
Ok, done